docs/applications/add-application.md

```diff
@@ -5,7 +5,9 @@ links:
     overview:
     quickstart:
     previous:
+        - /docs/ssh-keys/test-ssh-connection
     next:
+        - /docs/pipelines/add-new-step
```

Comment:
`add new` seems a bit redundant here, as the item to be added is always a new item.
I believe `<operation>-<resource type>` could be a simplified naming convention and it is already adopted in most of the existing docs files:
ted for 
```suggestion
        - /docs/pipelines/add-pipeline-step
```